claims_hosp/delphi_claims_hosp/run.py

```diff
@@ -114,12 +128,18 @@ def run_module(params):
                 signal_name
             )
             updater.update_indicator(
-                params["indicator"]["input_file"],
+                claims_file,
                 params["common"]["export_dir"],
                 logger,
             )
         logger.info("finished updating", geo = geo)
 
+    # Remove all the raw files
+    for fn in os.listdir(params["indicator"]["input_dir"]):
+        if ".csv.gz" in fn:
+            os.remove(f'{params["indicator"]["input_dir"]}/{fn}')
+    logger.info('Remove all the raw files.')
+
     elapsed_time_in_seconds = round(time.time() - start_time, 2)
     logger.info("Completed indicator run",
         elapsed_time_in_seconds = elapsed_time_in_seconds)
```

Comment:
In order to fit into the monitoring dashboard, we need this message to have the following information:
```suggestion
    logger.info("Completed indicator run",
                elapsed_time_in_seconds = elapsed_time_in_seconds,
                csv_export_count = ??, 
                max_lag_in_days = ??, 
                oldest_final_export_date = ??,
    )
```

See [the template run.py file](https://github.com/cmu-delphi/covidcast-indicators/blob/main/_template_python/delphi_NAME/run.py) for more information on how each element is defined and/or how to compute it.

Reply:
@krivard Updated with those variables just now